packages/cli/__test_scripts__/common.sh

```
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# common.sh
+# Common utilities for CLI test scripts
+
+# Set ELIZAOS_CMD based on IS_NPM_TEST environment variable
+# IS_NPM_TEST=false (default): Use locally built CLI
+# IS_NPM_TEST=true: Use global elizaos command
+setup_elizaos_cmd() {
+    local script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)